src/lib/Group.ts

```diff
@@ -76,7 +79,10 @@ export class Group<
     }
   }
 
-  async messages(): Promise<DecodedMessage<ContentTypes>[]> {
+  async messages(skipSync = false): Promise<DecodedMessage<ContentTypes>[]> {
```

Comment:
Thoughts on using an object instead?
Could get confusing especially when adding pagination and sorting later on

Reply:
Yea I agree, once we have pagination and sorting, passing in an object would be more straightforward. My inclination is to wait until one of those is ready before adding here, so I'll merge for now with the idea we'll update when those are ready in upcoming PR 🙏